examples/column_transformer.py

```diff
@@ -104,21 +65,22 @@ def fit(self, x, y=None):
         return self
 
     def transform(self, posts):
-        features = np.recarray(shape=(len(posts),),
-                               dtype=[('subject', object), ('body', object)])
+        # construct object dtype array with two columns
+        # first column = 'subject' and second column = 'body'
+        features = np.empty(shape=(len(posts), 2), dtype=object)
```

Comment:
In this example I didn't use pandas, as it seems a bit overhead (it would just be for temporarily putting the two columns in a frame to pass it to a next frame). But we certainly need another example with a pandas dataframe (eg with adults). 
But can change it here as well if needed.